src/new/Io/JLD2.jl

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
Puedes documentar con un poco mas de detalle que es `dataset` ?

Reply:
in HDF5 datasets are the differents groups of data. Each dataset has its own name in order to identify it. The dataset name in translated to a HDF5 path inside the file. 

Is an optional argument, the user/developer does need to know it (as default is "data"). But if he/she wants to store the data in a specific pach inside the HDF5/JLD2 file (e.g. `/gridap/mesh` instead of `/data`) it's possible.

I will add some extra docstrings, something like:
"Dataset: specify the data name and root location of the data inside the generated JLD2 file"

Reply:
Done in 36cc7c3